### DISCUSSION TITLE
Logging pc usage activity on Google Calendar

### DISCUSSION THREAD

**Discussion by alok8bb** [Waishnav/Watcher#9] [Ideas]
I've been tracking sleep with a sleep tracking application which creates a google calendar block of time every time it logs sleep. I wish there was same feature in an app that just tracks how many hours have you used pc (without logging actual apps and stuff) and then updates it on Google Calendar so that at the end of the day I can check how much time did I spend on pc just like Digital Wellbeing on phone.

Is it a good idea to implement this in watcher?

**Comment by Waishnav** [comment]
I think it's not that good to use the feature for all users, but I will think of it for the GUI version
